doc/examples/viz_slice.py

```diff
@@ -148,8 +148,85 @@
    :align: center
 
    **Simple slice viewer with an HSV colormap**.
+"""
+
+"""
+Now we would like to add the ability to click on a voxel and show its value
+on a panel in the window. The panel is a widget which requires access to
+different areas of the visualization pipeline and therefore we don't recommend
+using it with ``window.show``. The more appropriate way is to use the
+``ShowManager`` object, which allows accessing the pipline in different areas.
+"""
+
+show_m = window.ShowManager(renderer, size=(1200, 900))
+show_m.initialize()
+
+"""
+We'll start by creating the panel and adding it to the ``ShowManager``
+"""
+
+
+def build_textblock(text):
+    """Generate a textblock for the panel."""
+    textblock = ui.TextBlock2D()
+    textblock.message = text
+    textblock.font_size = 18
+    textblock.font_family = 'Arial'
+    textblock.justification = 'left'
+    textblock.bold = False
+    textblock.italic = False
+    textblock.shadow = False
+    textblock.actor.GetTextProperty().SetBackgroundColor(0, 0, 0)
+    textblock.actor.GetTextProperty().SetBackgroundOpacity(0.0)
+    textblock.color = (1, 1, 1)
+
+    return textblock
+
+label_position = build_textblock(text='Position:')
+label_value = build_textblock(text='Value:')
+
+result_position = build_textblock(text='')
+result_value = build_textblock(text='')
+
+panel_picking = ui.Panel2D(center=(200, 120),
+                           size=(250, 125),
+                           color=(0, 0, 0),
+                           opacity=0.75,
+                           align="left")
 
+panel_picking.add_element(label_position, 'relative', (0.1, 0.55))
+panel_picking.add_element(label_value, 'relative', (0.1, 0.25))
 
+panel_picking.add_element(result_position, 'relative', (0.45, 0.55))
+panel_picking.add_element(result_value, 'relative', (0.45, 0.25))
+
+show_m.ren.add(panel_picking)
+
+"""
+Add a left-click callback to the slicer. Also disable interpolation so you can
+see what you are picking.
+"""
+
+
+def left_click_callback(obj, ev):
+    """Get the value of the clicked voxel and show it in the panel."""
+    event_pos = show_m.iren.GetEventPosition()
+
+    obj.picker.Pick(event_pos[0],
+                    event_pos[1],
+                    0,
+                    show_m.ren)
+
+    i, j, k = obj.picker.GetPointIJK()
+    result_position.message = '(' + str(i) + ', ' + str(j) + ', ' + str(k) + ')'
```

Comment:
I propose you to use `format` :     `result_position.message = '({0},{1},{2})'.format(str(i), str(j), str(k)) `

What do you think ?

Reply:
Agreed

Reply:
Addressed in f211dca